src/Swashbuckle.AspNetCore.Filters/Examples/ExamplesConverter.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.OpenApi.Any;
+using Microsoft.OpenApi.Models;
+using Newtonsoft.Json;
+using Swashbuckle.AspNetCore.Filters.Extensions;
+
+namespace Swashbuckle.AspNetCore.Filters
+{
+    internal class ExamplesConverter
+    {
+        private readonly JsonFormatter jsonFormatter;
+        private readonly MvcOutputFormatter mvcOutputFormatter;
+        private readonly JsonSerializerSettings serializerSettings;
+
+        public ExamplesConverter(JsonFormatter jsonFormatter, MvcOutputFormatter mvcOutputFormatter, JsonSerializerSettings serializerSettings)
+        {
+            this.jsonFormatter = jsonFormatter;
+            this.mvcOutputFormatter = mvcOutputFormatter;
+            this.serializerSettings = serializerSettings;
+        }
+
+        public IOpenApiAny SerializeExampleXml(object exampleValue)
+        {
+            return new OpenApiString(exampleValue.XmlSerialize(mvcOutputFormatter));
+        }
+
+        public IOpenApiAny SerializeExampleJson(object exampleValue)
+        {
+            return new OpenApiRawString(jsonFormatter.FormatJson(exampleValue, serializerSettings));
+        }
+
+        private static IDictionary<string, OpenApiExample> ToOpenApiExamplesDictionary(
+            IEnumerable<ISwaggerExample<object>> examples,
+            Func<object, IOpenApiAny> exampleConverter)
+        {
+            var groupedExamples = examples.GroupBy(
```

Comment:
We can't use only `ToDictionary` metod, because it throws non-informative exception when user accidentally provides examples with duplicated names. Or maybe we should just throw in this case?

Reply:
I think that should 😃 
...but something informative.